tests/samples/OpaqueTypes.scala

```diff
@@ -11,8 +11,16 @@ object OpaqueTypes {
   type struct_point = native.CStruct2[native.CInt, native.CInt]
   type union_u = native.CArray[Byte, native.Nat._4]
   type u = union_u
+  type struct_structWithPointerToUndefinedStruct = native.CStruct1[native.Ptr[Byte]]
+  type union_unionWithPointerToUndefinedStruct = native.CArray[Byte, native.Nat._8]
+  type aliasToPointerOfUndefinedUnion = native.Ptr[Byte]
+  type functionPointerWithPointerToOpaqueType = native.CFunctionPtr1[native.Ptr[native.Ptr[Byte]], native.Ptr[Byte]]
   def move(point: native.Ptr[struct_point], x: native.CInt, y: native.CInt): native.Ptr[struct_point] = native.extern
   def processPoints(p: native.Ptr[points]): native.Ptr[union_u] = native.extern
+  def usePointerToUndefinedStruct(anonymous0: native.Ptr[Byte]): Unit = native.extern
```

Comment:
Looking at this I think it would be better to have

```scala

type struct_structWithPointerToUndefinedStruct = native.CStruct0
type union_unionWithPointerToUndefinedStruct = native.CArray[Byte, native.Nat._0] // not sure if 0 length is supported

def usePointerToUndefinedStruct(anonymous0: native.Ptr[struct_structWithPointerToUndefinedStruct]): Unit = native.extern
```

So `CStruct0` for opaque structs and then use the type alias in the pointers. With default opaque reprensentations  for structs and unions it should be possible to remove most use of replaceType except for updates to structs and unions.

Reply:
No helper class should be generated for such types.

Reply:
Seems like a good solution.
Should we leave a comment beside `CStruct0` that it is an incomplete type or opaque type, so no one will think that it is indeed empty structure?